4.gene-expression-signatures/scripts/3.latent_dim_plot.r

```diff
@@ -0,0 +1,76 @@
+
+suppressPackageStartupMessages(library(dplyr))
+suppressPackageStartupMessages(library(ggplot2))
+suppressPackageStartupMessages(library(cowplot))
+suppressPackageStartupMessages(library(reticulate))
+
+
+
+
+#Load data
+data_dir <- file.path("../4.gene_expression_signatures/results")
+
+results_file <- file.path(data_dir, "combined_z_matrix_gsea_results.csv")
```

Comment:
Should this file use Parquet as well? I only ask due to how the other data are exported. Feel free to dismiss if this is irrelevant here.

Reply:
Yep absolutely, thank you!